costajohnt/w03-d01homework.txt

```diff
@@ -0,0 +1,13 @@
+Get all icecream flavors “/icecream”
+
+Get icecream by ID “/icecream/:id”
+
+Get icecream fruity “/fruity/:id/icecream”
+
+Get ice-cream gluten free “/glutenfree/:id/icecream”
```

Comment:
what would `/glutenfree/3/pasta` do?  
Why not `/icecream/glutenfree`?  Which do you feel is a better way to categorize and why?